core-plugins/docs/SSHAction-action.md

```diff
@@ -21,15 +21,16 @@ Properties
 
 **user:** The username used to connect to host.
 
-**privateKeyFile:** The file path to Private key.
+**privateKeyLookupKey:** Key used to lookup private key from secure storage.
```

Comment:
Now it has 2 keys :) Actually, is it mandatory that this will always be a macro? If someone doesn't care about security, couldn't they just dump their private key in a textarea? I think we can just call it ``privateKey``. The description can be something like - "The private key to be used to perform the secure shell action. Users can also specify a key that will be looked up from the secure key management store in CDAP". However, in that case you'll need a way to distinguish between whether its a secure key or just text.